### PR TITLE
feat: Added verify variable to allow connection to custom certificates

### PR DESCRIPTION
**Pull request by andreasunique**

### CODE REVIEW

File: unique_sdk/unique_sdk/_http_client.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 except ImportError:
     requests = None
 
-from unique_sdk import _error
+from unique_sdk import _error, unique_sdk
```
Comment (sheiksadique) [inline unique_sdk/unique_sdk/_http_client.py:10]: This looks wrong. I think you want a separate import. 
```suggestion
import unique_sdk
from unique_sdk import _error
```

File: unique_sdk/unique_sdk/_http_client.py [hunk 2]
```diff
@@ -47,6 +47,9 @@ def _request_internal(self, method, url, headers, post_data, is_streaming):
         if getattr(self._thread_local, "session", None) is None:
             self._thread_local.session = self._session or self.requests.Session()
 
+        verify = unique_sdk.api_verify_mode
+
+        print(f"verify: {verify}")
```
Comment (unique-fabian) [inline unique_sdk/unique_sdk/_http_client.py:52]: Remove print

```suggestion
```

File: unique_sdk/unique_sdk/_http_client.py [hunk 3]
```diff
@@ -56,6 +59,7 @@ def _request_internal(self, method, url, headers, post_data, is_streaming):
                     data=json.dumps(post_data),
                     timeout=self._timeout,
                     **kwargs,
```
Comment (unique-fabian) [inline unique_sdk/unique_sdk/_http_client.py:61]: Flip verify and **kwargs

```suggestion
                    verify=verify,
                    **kwargs,
```

**Review by unique-fabian — APPROVED** [review]
lgtm. changes were applied

**Pull request merged**